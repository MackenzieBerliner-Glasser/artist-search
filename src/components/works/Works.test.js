import React from 'react';
import Works from './Works';
import { shallow } from 'enzyme';
import { MemoryRouter } from 'react-router-dom';

describe('Work', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Works works={[]} />);
  });

  it('renders Works', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('displays song title', () => {
    const wrapperWithRouter = shallow(
      <MemoryRouter>
        <Works works={[{ id: '12',  title: 'a song' }]} />
      </MemoryRouter>
    );
    expect(wrapperWithRouter.html()).toContain('a song');
  });
});
