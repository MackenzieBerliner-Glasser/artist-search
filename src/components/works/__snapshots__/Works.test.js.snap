// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Work renders Works 1`] = `<Fragment />`;
